goth_tests/pyproject.toml

```diff
@@ -23,7 +23,7 @@ pytest-asyncio = "^0.14"
 # goth = "^0.7"
 # to use development goth version uncomment below
 # goth = { git = "https://github.com/golemfactory/goth.git", branch = "fix-flaky-tests-wait_for_proposals" }
-goth = { git = "https://github.com/golemfactory/goth.git", rev = "b5b4ad3c4156899781888f12c9e614ad7f734af9" }
+goth = { git = "https://github.com/golemfactory/goth.git", branch = "master" }
```

Comment:
I think we should reference specific commit on master

Reply:
the lock file already references a specific commit, but can update this :)